### DISCUSSION TITLE
Search SMILES by Zinc ID on SDF File

### DISCUSSION THREAD

**Discussion by IchiruTake** [rdkit/rdkit#3829] [Q&A]
This is my sample SDF file. I wonder if there are any ways to help me search the SMILES via ZINC ID and read this file.  Moreover, as the SMILES is not been written, I cannot convert them at ChemUtils.SDFToCsv or PandaTools. Is there anyway to help out. The ZINC ID for this file is ZINC00000868. Thank you.
Note: I have over 4000+ files which I don't want to use hand-by-hand converter

bond_energy_mol/ZINC00000868.out
 OpenBabel01301309493D

 22 23  0  0  0  0  0  0  0  0999 V2000
   -2.2338    0.4052   -0.7649 C   0  0  0  0  0  0  0  0  0  0  0  0
   -1.2235   -0.2766    0.1315 C   0  0  0  0  0  0  0  0  0  0  0  0
   -1.4793   -0.6721    1.2550 O   0  0  0  0  0  0  0  0  0  0  0  0
    0.0463   -0.4043   -0.4369 N   0  0  0  0  0  0  0  0  0  0  0  0
    1.1280   -1.0152    0.2012 C   0  0  0  0  0  0  0  0  0  0  0  0
    2.4612   -0.6855    0.0312 C   0  0  0  0  0  0  0  0  0  0  0  0
    3.3647    0.5037   -0.8666 S   0  0  0  0  0  0  0  0  0  0  0  0
    5.2449   -0.1843   -0.1886 S   0  0  0  0  0  0  0  0  0  0  0  0
    4.6427   -1.4742    0.8161 C   0  0  0  0  0  0  0  0  0  0  0  0
    3.2737   -1.6003    0.8208 C   0  0  0  0  0  0  0  0  0  0  0  0
    2.4199   -2.4835    1.4458 N   0  0  0  0  0  0  0  0  0  0  0  0
    1.0813   -2.2164    1.0775 C   0  0  0  0  0  0  0  0  0  0  0  0
    0.1484   -2.9364    1.4096 O   0  0  0  0  0  0  0  0  0  0  0  0
    2.7958   -3.5828    2.2891 C   0  0  0  0  0  0  0  0  0  0  0  0
   -3.1753    0.5518   -0.2291 H   0  0  0  0  0  0  0  0  0  0  0  0
   -1.8645    1.3817   -1.1034 H   0  0  0  0  0  0  0  0  0  0  0  0
   -2.4312   -0.2047   -1.6556 H   0  0  0  0  0  0  0  0  0  0  0  0
    0.2622    0.1593   -1.2727 H   0  0  0  0  0  0  0  0  0  0  0  0
    5.3311   -2.1099    1.3782 H   0  0  0  0  0  0  0  0  0  0  0  0
    3.2915   -3.2316    3.2130 H   0  0  0  0  0  0  0  0  0  0  0  0
    1.8809   -4.1293    2.5575 H   0  0  0  0  0  0  0  0  0  0  0  0
    3.4895   -4.2688    1.7696 H   0  0  0  0  0  0  0  0  0  0  0  0
  1 15  1  0  0  0  0
  1  2  1  0  0  0  0
  2  3  2  0  0  0  0
  4  2  1  0  0  0  0
  4  5  1  0  0  0  0  123.23
  5 12  1  0  0  0  0
  6  5  1  0  0  0  0
  6 10  2  0  0  0  0
  7  8  1  0  0  0  0
  7  6  1  0  0  0  0
  8  9  2  0  0  0  0
  9 10  1  0  0  0  0
  9 19  1  0  0  0  0
 10 11  1  0  0  0  0
 11 14  1  0  0  0  0
 12 13  2  0  0  0  0
 12 11  1  0  0  0  0
 14 21  1  0  0  0  0
 14 20  1  0  0  0  0
 16  1  1  0  0  0  0
 17  1  1  0  0  0  0
 18  4  1  0  0  0  0
 22 14  1  0  0  0  0
M  RAD  1   5   2
M  END
$$$$

**Comment by rajathsalegame** [comment]
I'm not sure if there's a way to get to get around the SMILES without reading in the sdf file first...
